doc/source/tutorial-quickstart-mlx.rst

```diff
@@ -5,21 +5,28 @@ Quickstart MLX
 ==============
 
 
-In this tutorial we will learn how to train simple MLP on MNIST using Flower and MLX.
-
-First of all, it is recommended to create a virtual environment and run everything within a :doc:`virtualenv <contributor-how-to-set-up-a-virtual-env>`.
+In this federated learning tutorial we will learn how to train simple MLP on MNIST using Flower and MLX. It is recommended to create a virtual environment and run everything within a :doc:`virtualenv <contributor-how-to-set-up-a-virtual-env>`.
 
 Let's use `flwr new` to create a complete Flower+MLX project. It will generate all the files needed to run, by default with the Simulation Engine, a federation of 10 nodes using `FedAvg <https://flower.ai/docs/framework/ref-api/flwr.server.strategy.FedAvg.html#flwr.server.strategy.FedAvg>`_. The dataset will be partitioned using Flower Dataset's `IidPartitioner <https://flower.ai/docs/datasets/ref-api/flwr_datasets.partitioner.IidPartitioner.html#flwr_datasets.partitioner.IidPartitioner>`_.
 
-Now that we have a rough idea of what this example is about, let's get started. We first need to create an MLX project. You can do this by running the command below. You will be prompted to give a name to your project as well as typing your developer name.:
+Now that we have a rough idea of what this example is about, let's get started. First, install Flower in your new environment:
+
+.. code-block:: shell
+
+    # In a new Python environment
+    $ pip install flwr
```

Comment:
```suggestion
Now that we have a rough idea of what this example is about, let's get started. First, install Flower in your new virtual environment:

.. code-block:: shell

    $ pip install flwr[simulation]
```

Reply:
I'd say we don't need to install the simulation extra here (it will anyway when installing the package).